pages/_app.js

```diff
@@ -1,24 +1,32 @@
 import GlobalStyles from '../components/GlobalStyles';
 import Head from 'next/head';
-import useLocalStorageState from 'use-local-storage-state';
-import { dummyitems } from '../lib/data';
-import { dummystorage } from '../lib/storageData';
 import { differenceInCalendarDays } from 'date-fns';
+import { useEffect, useState } from 'react';
 
 function MyApp({ Component, pageProps }) {
-  const [items, setItems] = useLocalStorageState('items', {
-    defaultValue: dummyitems,
-  });
-  const [storages, setStorages] = useLocalStorageState('storages', {
-    defaultValue: dummystorage,
-  });
-  function handleStorage(storage) {
-    setStorages(storage);
+  const [items, setItems] = useState([]);
+  const [storages, setStorages] = useState([]);
+
+  async function getItems() {
```

Comment:
maybe you could name this "handleGetItems" because it's passed down with "onGetItems"